servers/zms/src/main/java/com/yahoo/athenz/zms/store/AthenzDomain.java

```diff
@@ -28,16 +30,18 @@
     private List<Group> groups;
     private List<Policy> policies;
     private List<ServiceIdentity> services;
+    private Map<String, StringList> tags;
```

Comment:
why did we add tags as a field here? The domain object below already has the tags as a field since it's part of the meta data so I would expect that the domain object contains the tags when the AthenzDomain object was created thus there is no need for this extra field here.

Reply:
indeed, initially I didn't notice the domain object is included there.
Will fix it